system_strings/sysmes_text.en

```diff
@@ -814,77 +814,77 @@ I Kiss the Vampire Princess
 Mystic Eyes of Death Perception
 The curtain falls.
 Krake IV
-15日目・朝。
-籠城
-琥珀を追い返す。
-食事だけ置いていってもらう。
-決壊
-逃げ出した先に。
-地の底に来たれ、断罪者
-前方に逃げる。
-後方に逃げる。
-静かの海
-……眼鏡を外す。
-……眼鏡は外さない。
-復讐者
-フランス事変
-薔薇園からの生還
-最短で、あの吸血鬼を殺す。
-シエル先輩を連れて逃げる。
-ウサギ狩り
-面倒だ。殺してしまえ。
-………………いや、それは。
-抗戦
-ルナ・ボウ
-クリスマス・チャペル
-灰は灰に
-不可視の花
-いっときの休息
-気になるけどスルー。
-スルーできない。
-健やかなる時も
-夜明け前
+15th day ― Morning
+Besieged
+Drive away Kohaku
+Just leave the food there.
+Breach
+Before escaping
+Judge in the other depths
+Escape forward.
+Escape back.
+Quiet ocean
+Remove my glasses.
+Don’t remove my glasses.
+Avenger
+The French Incident
+Homecoming from the rose garden
+Kill that vampire at close range.
+Escape with Ciel-senpai
+Rabbit Hunting
+It’s a bother. Kill it.
+No, that would be...
+Resistance.
+Lunar Bow
+Christmas Chapel
+Ashes to ashes
+Invisible flower
+Brief reprieve
+I’m curious, but I’ll let it go.
+I can’t let it go.
+When I get better, too
+Before daybreak.
 Moon Princess
-……アルクェイドには従えない。
-真祖堕天
-明日になったら。
-世界の死
-決別
-名残の夢
-嫌いにはなれない。
-事象収納
-世界の悲鳴
-狙撃
-着弾
-ロアの声
-180秒臨界決戦
-クラーケ　Ⅲ
-白亜の雪
-顕現
-最後の警告
-それがどうした、黙ってろ。
-じゃあ、どうしろって言うんだ。
+Don’t accompany Arcueid.
+Fallen True Ancestor
+If tomorrow comes.
+Death into the world.
+Farewell
+The remains of the dream
+I can’t come to dislike it.
+Event Storage
+Cries of the planet.
+Sniping
+Impact
+The voice of Roa
+180 second decisive barrier fight
```

Comment:
sounds like complete ass and makes 0 sense, but i didnt find a great way to fit that much information in so few characters